app/packs/entrypoints/decidim_application.js

```diff
@@ -1,5 +1,7 @@
 // This file is compiled inside Decidim core pack. Code can be added here and will be executed
 // as part of that pack
+import "src/decidim/admin/process"
+import "src/decidim/participa/regulations"
```

Comment:
until now we had a bug because these files were being ignored? just to understand..

Reply:
Yes, there is a bug...